src/patternfly/components/Alert/alert.scss

```diff
@@ -2,216 +2,172 @@
   // Component variables
   --pf-c-alert--BoxShadow: var(--pf-global--BoxShadow--lg);
   --pf-c-alert--BackgroundColor: var(--pf-global--BackgroundColor--100);
-  --pf-c-alert--grid-template-columns: max-content 1fr max-content;
-  --pf-c-alert--grid-template-rows: 1fr auto;
+  --pf-c-alert--GridTemplateColumns: max-content 1fr max-content;
+  --pf-c-alert--before--BorderTopWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-alert--before--BorderTopColor: var(--pf-global--default-color--200);
+  --pf-c-alert--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-alert--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-alert--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-alert--PaddingLeft: var(--pf-global--spacer--md);
 
   // Icon
-  --pf-c-alert__icon--Padding: var(--pf-global--spacer--md);
-  --pf-c-alert__icon--Color: var(--pf-global--default-color--300);
-  --pf-c-alert__icon--BackgroundColor: var(--pf-global--default-color--100);
-  --pf-c-alert__icon--FontSize: var(--pf-global--icon--FontSize--lg);
+  --pf-c-alert__icon--Color: var(--pf-global--default-color--200);
+  --pf-c-alert__icon--MarginTop: calc(var(--pf-global--spacer--xs) / 2);
+  --pf-c-alert__icon--MarginRight: var(--pf-global--spacer--xs);
+  --pf-c-alert__icon--FontSize: var(--pf-global--icon--FontSize--md);
 
   // Title
   --pf-c-alert__title--FontSize: var(--pf-global--FontSize--md);
```

Comment:
Can you think of a reason to set this if it's the default font size?

Reply:
I was using it to calculate the height, i dont need anymore good pick up